### DISCUSSION TITLE
Subdomain automatically redirects to domain

### DISCUSSION THREAD

**Discussion by Mathiaszero** [caprover/caprover#2074] [General]
I have a domain in one ip address/VPS on caprover and I recently changed it to a subdomain. However after I changed it and added the subdomain to the A record it redirects to the domain automatically, when it shouldnt . How can I resolve this issue?

**Comment by githubsaturn** [comment]
Redirects doesn't happen on CapRover. Is this a dashboard or an app that is deployed on CapRover, e.g. wordpress etc?

**Reply by Mathiaszero** [reply]
Wordpress, to answer your question. I have decided to suspend the project I am working on so am closing this for now. Thanks for assisting.

**Reply by githubsaturn** [reply]
For future reference, this is not related to CapRover, WordPress has internal redirect mechanism you should change the URL in their settings

![image](https://github.com/caprover/caprover/assets/10582082/3d8963bf-cca2-4b39-893e-cd2b51d61ab8)
